compiler/quilt/__init__.py

```diff
@@ -119,3 +119,5 @@ def handle_interrupt(signum, stack):
     version_add,
     version_list,
 )
+
+from .tools.util import save
```

Comment:
A blank line at the very end of each file should eliminate the "No EOF" warning we see above